backend/Tim.Backend/Startup/Config/KustoConfiguration.cs

```diff
@@ -22,16 +22,16 @@ public KustoConfiguration()
         }
 
         /// <summary>
-        /// Gets or sets the kusto query url for the main service cluster.
+        /// Gets or sets the kusto query ingest url for the main service cluster.
         /// </summary>
         [Required]
-        public string KustoClusterUri { get; set; } = Environment.GetEnvironmentVariable("KUSTO_URL");
+        public string IngestKustoClusterUri { get; set; } = Environment.GetEnvironmentVariable("KUSTO_INGEST_URL");
```

Comment:
why remove the kusto cluster uri? isnt that also needed?

Reply:
Usually, the user provides the cluster. However, I do end up needing it for control commands in some code I'm writing now.